saturn-gpu/environment.yml

```diff
@@ -16,16 +15,20 @@ dependencies:
 - pandas
 - ipywidgets
 - matplotlib
-- dask=2.12.0
-- distributed=2.12.0
-- dask-saturn=0.0.3
-- pip
+- dask=2.20.0
+- distributed=2.20.0
+- dask-saturn=0.0.4
+- xgboost
```

Comment:
An issue just came up where we should pin xgboost to 0.9 for use with dask-xgboost.

Reply:
Chose not to include pin because latest release of `dask-xgboost` now pins `xgboost`. Plus I think we don't want to maintain too much specific use case stuff in these images https://github.com/dask/dask-xgboost/issues/75#issuecomment-668585832